src/interface/web/app/factchecker/page.tsx

```diff
@@ -533,7 +533,7 @@ export default function FactChecker() {
                         <Button disabled={clickedVerify} onClick={() => onClickVerify()}>Verify</Button>
                     </div>
                     <h3 className={`mt-4 mb-4`}>
-                        Try with a particular model. You must be <a href="/config" className="font-medium text-blue-600 dark:text-blue-500 hover:underline">subscribed</a> to configure the model.
+                        Try with a particular model. You must be <a href="/configure" className="font-medium text-blue-600 dark:text-blue-500 hover:underline">subscribed</a> to configure the model.
```

Comment:
I think we should also change the page to be called settings. In `web_client.py`, we can temporarily add a redirect for people navigating to `/config` to go to `/settings`/.

Reply:
Yeah, I'd been shuttling between calling it configure vs settings. Configure makes it action oriented. Settings is more standard. I've renamed the user facing pages across web, desktop to use `Settings`. The API is still called `/api/configure` as the act of calling the API can be named with more action oriented semantics